app/models/user-store/user-store.ts

```diff
@@ -218,12 +253,16 @@ export const UserStoreModel = types
           email,
           avatarURL,
           isCivicLiker,
+          likeWallet,
+          cosmosWallet,
         })
       } else if (self.currentUser) {
         self.currentUser.displayName = displayName
         self.currentUser.email = email
         self.currentUser.avatarURL = avatarURL
         self.currentUser.isCivicLiker = isCivicLiker
+        self.currentUser.likeWallet = likeWallet
+        self.currentUser.cosmosWallet = cosmosWallet
```

Comment:
What is the use for `cosmosWallet` now?

Reply:
Maybe in some parts of the legacy code
Should clean up with another PR